packages/Search/src/details/DTK_DetailsTreeVisualization.hpp

```diff
@@ -0,0 +1,144 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+#ifndef DTK_DETAILS_TREE_VISUALIZATION_HPP
+#define DTK_DETAILS_TREE_VISUALIZATION_HPP
+
+#include <DTK_DetailsTreeTraversal.hpp>
+
+#include <Kokkos_View.hpp>
+
+#if !defined( ARBORX_ENABLE_VIZ )
+static_assert(
+    false,
+    "do not include this header when tree visualization mode is not enabled" );
```

Comment:
Why?

Reply:
Cause signature of distance function is different.  Anyway this is removed in the changes I am about to push.